src/helpers.ts

```diff
@@ -77,3 +77,28 @@ export function setBody(body: string) {
 export function setNamespace(namespace: string) {
   setAttribute("appsignal.namespace", namespace)
 }
+
+export function setError(error: Error) {
+  if (error && error.name && error.message) {
+    const activeSpan = trace.getActiveSpan()
+    if (activeSpan) {
+      activeSpan.recordException(error)
+      activeSpan.setStatus({
+        code: SpanStatusCode.ERROR,
+        message: error.message
+      })
+    }
+  }
+}
+
+export function sendError(error: Error, fn: () => void = () => {}) {
+  if (error && error.name && error.message) {
+    trace
+      .getTracer("Appsignal.sendError")
+      .startActiveSpan(error.name, { root: true }, span => {
```

Comment:
The changing of the active span doesn't break anything down the line? Like if you send an error this way and then want to continue working with the previous active span, that will work?

For the AppSignal span and transaction APIs I think it "broke" the current span if we did the flow like this with changing the active span.

Reply:
I think not -- OpenTelemetry docs say that the span will be active _for the duration of the span_ -- but I'll add a test to check.